src/NuGet.Core/NuGet.Packaging/PackageFolderReader.cs

```diff
@@ -68,7 +70,14 @@ public override string GetNuspecFile()
 
             if (nuspecFiles.Length == 0)
             {
-                throw new PackagingException(Strings.MissingNuspec);
+                throw new PackagingException(NuGetLogCode.NU5037, string.Format(
```

Comment:
can we do this with a single `string.Format`? Or use a StringBuilder?

Reply:
Browsed through the source code of [String.Format](https://referencesource.microsoft.com/#mscorlib/system/string.cs,bdf91919a8d3537e).It looks like String.Format method internally refers to StringBuilder.AppendFormat method. Modified code to use one instance of StringBuilder instead of two calls to String.Format method.